packages/@o3r/eslint-plugin/src/rules/json/json-dependency-versions-harmonize/json-dependency-versions-harmonize.ts

```diff
@@ -41,8 +41,7 @@ export default createRule<[Options, ...any], 'versionUpdate' | 'error'>({
     hasSuggestions: true,
     type: 'problem',
     docs: {
-      description: 'Ensure that the package dependency versions are aligned with the other package of the workspace.',
-      recommended: 'strict'
+      description: 'Ensure that the package dependency versions are aligned with the other package of the workspace.'
```

Comment:
```suggestion
      description: 'Ensure that the package dependency versions are aligned with the other packages of the workspace.'
```
or leave package singular depending on the use case?